recipes/config.rb

```diff
@@ -26,11 +26,3 @@
   variables lazy { node['stackstorm']['config'] }
   action :create
 end
-
-directory 'st2:log_dir' do
```

Comment:
Log dir is created by `.deb/.rpm` packages, not by this recipe.

But we'll definitely need checks like that (and much more) as part of Integration Tests https://github.com/StackStorm/chef-stackstorm/issues/29.